wtxmgr/tx.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/btcsuite/btcd/wire"
-	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcd/btcutil"
```

Comment:
`wtxmgr` has it's own go.mod to update this as well.

Reply:
Should be addressed now